.vib/gitlab-runner/goss/vars.yaml

```diff
@@ -0,0 +1,11 @@
+binaries:
+  - gitlab-runner
+files:
+  - paths:
+      - /opt/bitnami/gitlab-runner/bin/gitlab-runner
```

Comment:
You are implicitly checking this binary file existence in the above `binaries` section, so I don't think this adds much value.